scss/bitstyles/ui/forms.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="UI/Data/Forms" />
+
+# Forms
+
+**See inputs & labels for examples of using inputs of various types.**
+
+When using form elements in a form, they should always be wrapped in a list element, as this indicates to the user that all the elements are closely related to each other. This also gives us the opportunity to apply some layout, such as adding space between elements.
```

Comment:
Kinda! :) It’s using the html elements to describe the content. And it’s convenient for layout. Actually its something I’m keeping an eye on/reconsidering, because some screenreader users dislike the extra info it gives, and just want the inputs 🤔